setup/gcp/installer/forseti_installer.py

```diff
@@ -185,23 +212,24 @@ def wait_until_vm_initialized(self, vm_name):
         """
 
         installation_type = self.config.installation_type
-        utils.print_banner('{} VM Initialization'.format(
+        utils.print_banner('Forseti {} VM Initialization'.format(
             installation_type.capitalize()))
         _, zone, name = gcloud.get_vm_instance_info(vm_name)
 
         status_checker = (lambda: gcloud.check_vm_init_status(name, zone))
 
-        loading_message = ('This may take a few minutes. Waiting for Forseti '
-                           '{} to be initialized..'.format(installation_type))
+        loading_message = ('Waiting for Forseti {} to be initialized..'.format(
```

Comment:
nit: Just be consistent `... `, e.g. three dots and a space on loading messages. Fix throughout.

Reply:
Updated to have 3 dots